awx/main/tests/functional/test_execution_environments.py

```diff
@@ -0,0 +1,15 @@
+import pytest
+from unittest import mock
+from rest_framework.exceptions import PermissionDenied
+
+from awx.main.models import (ExecutionEnvironment, Organization)
+
+
+@pytest.mark.django_db
+def test_execution_environment_creation(execution_environment, organization):
+    execution_env = ExecutionEnvironment.objects.create(name='Hello Environment', image='', organization=organization, managed_by_tower=False, credential=None, pull='missing')
```

Comment:
@rebeccahhh can one copy one EE `managed_by_tower`?

Reply:
@nixocio I'm not sure, @jbradberry do you know?

Reply:
@nixocio there's no rbac logic that prevents it.  The expectation is that (some) global EEs are going to be the ones that are `managed_by_tower=True`, but there's nothing currently stopping users from creating global ones without it or Org-attached ones with it.  Anyway, for the purposes of rbac, my expectation is that if you can see it you can copy it, as long as you have sufficient privileges to do so in the current context (e.g. you have an Org Admin role).

... that raises an interesting question:  should an Org Admin be able to copy a global EE into their Org, and have that Org attachment happen automatically as part of the copy operation?

Reply:
Thanks for your explanation @jbradberry. Regarding the last question an Org Admin can be the admin of more than Org, then automatically attaching the Org will not be a good fit, right? Right now for Org Admins to create an EE they must provide an Org.